woocommerce-services.php

```diff
@@ -45,7 +45,7 @@
 	define( 'WOOCOMMERCE_CONNECT_MAX_JSON_DECODE_DEPTH', 32 );
 
 	if ( ! defined( 'WOOCOMMERCE_CONNECT_SERVER_API_VERSION ' ) ) {
```

Comment:
This is entirely unrelated to your PR but it looks like a trailing space snuck into this line. Could you please remove it?

```suggestion
	if ( ! defined( 'WOOCOMMERCE_CONNECT_SERVER_API_VERSION' ) ) {
```